docs/reference/search/search-your-data/retrieve-selected-fields.asciidoc

```diff
@@ -539,95 +599,9 @@ GET /_search
 
 NOTE: <<source-filtering,`_source`>> and <<request-body-search-version, `version`>> parameters cannot be activated if `_none_` is used.
 
-[discrete]
-[[source-filtering]]
-=== Source filtering
-
-You can use the `_source` parameter to select what fields of the source are
-returned. This is called _source filtering_.
-
-The following search API request sets the `_source` request body parameter to
-`false`. The document source is not included in the response.
-
-[source,console]
-----
-GET /_search
-{
-  "_source": false,
-  "query": {
-    "match": {
-      "user.id": "kimchy"
-    }
-  }
-}
-----
-
-To return only a subset of source fields, specify a wildcard (`*`) pattern in
-the `_source` parameter. The following search API request returns the source for
-only the `obj` field and its properties.
-
-[source,console]
-----
-GET /_search
-{
-  "_source": "obj.*",
-  "query": {
-    "match": {
-      "user.id": "kimchy"
-    }
-  }
-}
-----
-
-You can also specify an array of wildcard patterns in the `_source` field. The
-following search API request returns the source for only the `obj1` and
-`obj2` fields and their properties.
-
-[source,console]
-----
-GET /_search
-{
-  "_source": [ "obj1.*", "obj2.*" ],
-  "query": {
-    "match": {
-      "user.id": "kimchy"
-    }
-  }
-}
-----
-
-For finer control, you can specify an object containing arrays of `includes` and
-`excludes` patterns in the `_source` parameter.
-
-If the `includes` property is specified, only source fields that match one of
-its patterns are returned. You can exclude fields from this subset using the
-`excludes` property.
-
-If the `includes` property is not specified, the entire document source is
-returned, excluding any fields that match a pattern in the `excludes` property.
-
-The following search API request returns the source for only the `obj1` and
-`obj2` fields and their properties, excluding any child `description` fields.
-
-[source,console]
-----
-GET /_search
-{
-  "_source": {
-    "includes": [ "obj1.*", "obj2.*" ],
-    "excludes": [ "*.description" ]
-  },
-  "query": {
-    "term": {
-      "user.id": "kimchy"
-    }
-  }
-}
-----
-
 [discrete]
 [[script-fields]]
-=== Script fields
+==== Script fields
```

Comment:
I'm tempted to rewrite this to just use `fields` with runtime fields.

Reply:
@nik9000, I talked to Luca about this the other day. My thought is that we could update this script fields section to suggest that runtime fields is a more flexible option, and to retrieve scripted results with the `fields` option. After 7.13 feature freeze, we can discuss the plan for scripted fields going forward and how we want to address that feature.

Reply:
That's fine with me too.